onboarding/source/amb_light_service.c

```diff
@@ -17,12 +26,23 @@
 
 /* USER CODE BEGIN */
 // Define light service queue config here
-
+#define LIGHT_QUEUE_LENGTH 10
+#define LIGHT_QUEUE_ITEM_SIZE sizeof(light_event_t)
+#define LIGHT_QUEUE_SIZE sizeof( uint64_t )
 /* USER CODE END */
 
 /* USER CODE BEGIN */
 // Declare any global variables here
 
+
+static TaskHandle_t lightServiceHandle = NULL;
+static StaticTask_t lightServiceBuffer;
+static StackType_t lightServiceStack[LIGHT_SERVICE_STACK_SIZE];
+
+static QueueHandle_t lightserviceQueue;
+static StaticQueue_t xlightStaticQueue;
+uint8_t uclightQueueStorageArea[LIGHT_QUEUE_LENGTH * LIGHT_QUEUE_SIZE];
```

Comment:
The size of this array should be able to store all the items in the queue. Each item in the queue is of type `light_event_t` and there are a maximum of `LIGHT_QUEUE_LENGTH` items. So the size of this array needs to be at least `LIGHT_QUEUE_LENGTH * LIGHT_QUEUE_ITEM_SIZE `